amass, httprobe, sublist3r, theHarvester: add page (#4912)
tldr-pages/tldr: pages/common/sublist3r.md

# sublist3r

> Fast subdomains enumeration tool for penetration testers.
> More information: <https://github.com/aboul3la/Sublist3r>.

- Find subdomains for a domain:

`sublist3r --domain {{domain_name}}`

> - Find subdomains for a domain, also use brute force:

Reviewer: Perhaps this will read better?
```suggestion
- Find subdomains for a domain, also enabling brute force search:
```

`sublist3r --domain {{domain_name}} --bruteforce`

> - Save found results to a text file:

Reviewer: Perhaps this will read better?
```suggestion
- Save the subdomains found to a text file:
```

> `sublist3r --domain {{domain_name}} --output {{output_file}}`

Reviewer: Perhaps this would be more consistent with other existing pages?
```suggestion
`sublist3r --domain {{domain_name}} --output {{path/to/output_file}}`
```
